react/features/conference/components/Conference.web.js

```diff
@@ -74,10 +84,13 @@ class Conference extends Component<Props> {
         APP.UI.registerListeners();
         APP.UI.bindEvents();
 
-        const { dispatch, t } = this.props;
+        const { _alwaysVisibleToolbar, dispatch, t } = this.props;
 
         dispatch(connect());
         maybeShowSuboptimalExperienceNotification(dispatch, t);
+
+        dispatch(setToolboxAlwaysVisible(
+            _alwaysVisibleToolbar || interfaceConfig.filmStripOnly));
```

Comment:
setToolboxAlwaysVisible didn't have to be the responsibility of the toolbar itself. It was easier to support both old and new toolbars by moving this logic out of toolbars and into a higher level.